geopandas/array.py

```diff
@@ -174,13 +230,18 @@ def from_wkb(data, crs=None):
     ----------
     data : array-like
         list or array of WKB objects
+        The 'srid' can be read from geoalchemy2's WKBElement.
     crs : value, optional
         Coordinate Reference System of the geometry objects. Can be anything accepted by
         :meth:`pyproj.CRS.from_user_input() <pyproj.crs.CRS.from_user_input>`,
         such as an authority string (eg "EPSG:4326") or a WKT string.
```

Comment:
```suggestion
        such as an authority string (eg "EPSG:4326") or a WKT string. If None, geopandas will try to infer CRS from the SRID of geometries (if available).
```

I would add this here, instead of that sentence above (note it will need better formatting to avoid linting fail).